site/student-lectures/Coevolution/Coevolution.rst

```diff
@@ -0,0 +1,219 @@
+**************************************************
+Coevolution in Evolutionary Computation
+**************************************************
+
+1. Introduction
+===============
+
+1.1 Definition of Coevolution
+-----------------------------
```

Comment:
you do not need to add the numbers; the numbers get added by sphinx

Reply:
done!